integration_tests/integration/bookAVisit.cy.ts

```diff
@@ -120,8 +120,8 @@ context('Book a visit', () => {
       cy.contains(contacts[0].restrictions[0].restrictionTypeDescription)
       cy.contains('End date not entered')
     })
-    selectVisitorsPage.getVisitor(contacts[0].personId).check()
-    selectVisitorsPage.getVisitor(contacts[1].personId).check()
+    selectVisitorsPage.getVisitor(contacts[0].personId).check({ force: true })
```

Comment:
govuk-frontend v5.1.0 has changed something (likely this https://github.com/alphagov/govuk-frontend/pull/4093?) with checkbox input/labels that makes Cypress think a element is hidden when it is not. Fix to force the `.check()`

Not really sure if it's an issue with Cypress or the new markup; but ideally we will be able to remove these overrides at some point....

Reply:
Hi @tpmcgowan, just spotted you had to do the same as us [here](https://github.com/ministryofjustice/hmpps-accredited-programmes-ui/pull/504). Just thought I'd mention that I've seen it's been sorted in 5.2.0 now so `force: true` can be removed.

Reply:
Thank-you for letting me know, @jsrobertson  🙂!